index.html

```diff
@@ -3817,122 +3817,93 @@ <h3>Introduction</h3>
+								><code>div</code></a> elements, for example, as the algorithm <a
+								href="#toc-ignored-elements">ignores elements</a> that are not relevant to its
+							processing. The list cannot occur inside of any <a href="#toc-skipped-elements">skipped
+								elements</a>, however, since their internal contents are not evaluated.</p>
+						<p>If the <code>nav</code> element does not contain one of these elements, then user agents will
+							not register the Web Publication as containing a usable table of contents (e.g., a
+							machine-rendered option will not be available).</p>
+					</dd>
+					<dt id="toc-branches">Branches</dt>
+					<dd>
+						<p>If the table of contents is considered as a tree of links, then each list item (<a
+								href="https://www.w3.org/TR/html/grouping-content.html#the-li-element"><code>li</code>
+								element</a>) inside of the <a href="toc-list-links">list of links</a> represents one
+							branch. Each of these branches has to have a name and optional destination in order to be
+							presented to users, and this information is obtained from the first <a
+								href="https://www.w3.org/TR/html/textlevel-semantics.html#the-a-element"><code>a</code>
+								element</a> found within the list item, <a href="#toc-ignored-elements">wherever it is
+								nested</a> (again, excluding any <code>a</code> elements inside of <a
+								href="toc-skipped-elements">skipped elements</a>.)</p>
+						<p>The link destination for the branch is obtained from the <code>a</code> element's
+								<code>href</code> attribute, when specified. This attribute can be omitted if a link is
+							not available (e.g., in a preview) or not relevant (e.g., a grouping header). When providing
+							a link into the content, it is also possible to specify the relationship of the linked
+							document (in a <code>rel</code> attribute) and the media type of the linked resource (in a
+								<code>type</code> attribute).</p>
+						<p>After finding the <code>a</code> element that labels the branch, user agents will continue to
+							inspect the markup for another list element (i.e., sub-branches). If a list is found, it is
+							similarly processed to extract its links, and so on, until there are no more nested branches
+							left to process.</p>
+					</dd>
+					<dt id="toc-skipped-elements">Skipped Elements</dt>
+					<dd>
+						<p>A small set of elements are ignored when the parsing table of contents to avoid
+							misinterpretation. These are the [[html]] <a
+								href="https://www.w3.org/TR/html/dom.html#sectioning-content">sectioning content
+								elements</a> and <a href="https://www.w3.org/TR/html/sections.html#sectioning-roots"
+								>sectioning root elements</a>. The reason they are ignored is because they can defined
+							their own outlines (i.e., they can represent embedded content that is self-contained and not
```

Comment:
... or any element whose <a href="https://www.w3.org/TR/html/editing.html#element-attrdef-global-hidden"><code>hidden</code></a> attribute is set.